specification/dartLangSpec.tex

```diff
@@ -8848,53 +8851,84 @@ \subsubsection{Unqualified Invocation}
 }
 
 \LMHash{}%
-It is a compile-time error if $i$ occurs inside a top-level or static function
-(be it function, method, getter, or setter)
-or a top-level or static variable initializer,
-and there is no lexically visible declaration named \id{} in scope.
+For the static analysis of $i$,
+a lexical lookup of \id{} is performed
+(\ref{lexicalLookup}):
 
 \LMHash{}%
-If there exists a lexically visible declaration named \id,
-let $D_{id}$ be the innermost such declaration.
-Then:
+\Case{Lexical lookup yields a declaration}
+In this case the lexical lookup of \id{}
```

Comment:
Is that a definition, an assertion or just unnecessary flavor text? It seems to just repeat the Case text, and it's unclear why it does that.

Maybe just:
> Let $D$ be the declaration yielded by the lexical lookup of \id{}.